osu.Game/Screens/Play/HUD/HitErrorMeters/ColourHitErrorMeter.cs

```diff
@@ -17,11 +18,36 @@ namespace osu.Game.Screens.Play.HUD.HitErrorMeters
 {
     public class ColourHitErrorMeter : HitErrorMeter
     {
-        internal const int MAX_DISPLAYED_JUDGEMENTS = 20;
-
+        private const int default_shape_alpha = 0;
         private const int animation_duration = 200;
         private const int drawable_judgement_size = 8;
-        private const int spacing = 2;
+
+        [SettingSource("Hit error amount", "Number of hit error shapes")]
+        public BindableNumber<int> HitShapeCount { get; } = new BindableNumber<int>(20)
+        {
+            MinValue = 1,
+            MaxValue = 30,
+            Precision = 1
+        };
+
+        [SettingSource("Opacity", "Visibility of object")]
```

Comment:
What's the use case for this? It's adjusting the opacity of only the judgements but not the surrounding display?

Would this not be better suited as a skin component wide adjustment that could be applied to any/every skin component?

Reply:
it would indeed be better if every component had its own alpha slider, i just did a basic implementation in the vein of my use of the bigblackbox test component, but ill remove this and make a pr sometime with alpha adjustments for ever object in a skin